pkg/proc/native/proc_windows.go

```diff
@@ -300,6 +301,10 @@ func (dbp *Process) waitForDebugEvent(flags waitForDebugEventFlags) (threadID, e
 		case _RIP_EVENT:
 			break
 		case _EXCEPTION_DEBUG_EVENT:
+			if flags&waitContinueAllExceptions != 0 {
+				continueStatus = _DBG_CONTINUE
+				break
```

Comment:
This is breaking out of the switch not out of the loop: https://play.golang.org/p/vsmUbcdqba